metpy/plots/station_plot.py

```diff
@@ -251,7 +251,7 @@ def plot_barb(self, u, v, **kwargs):
             The data to use for the u-component of the barbs.
         v : array-like
             The data to use for the v-component of the barbs.
-        plot_units: `pint.unit`
+        plot_units: `pint.Unit`
```

Comment:
We should probably standardize on one way, either with `` ` `` or without for function/method arguments. Not sure what best practice, right is.